RELEASES.md

```diff
@@ -1,3 +1,144 @@
+Version 1.19.0 (2017-07-20)
+===========================
+
+Language
+--------
+
+- [Numeric fields can now be used for creating tuple structs.][36868]
+  For example `struct Point(u32, u32); let x = Point { 0: 7, 1: 0 };`.
+- [Macro recursion limit increased to 1024 from 64.][41676]
+- [Added lint for detecting unused macros.][41907]
+- [`loop` can now return a value with `break`.][42016]
+  For example: `let x = loop { break 7; };`
+- [C compatible `union`s are now available.][42068] They can only contain `Copy`
+  types and cannot have a `Drop` implementation.
+  Example: `union Foo { bar: u8 }`
+
+Compiler
+--------
+
+- [Added bootstrap support for Android.][41370]
+- [Change `arm-linux-androideabi` to correspond to the `armeabi`
+  official ABI.][41656]
```

Comment:
It might be useful to note that developers must now use `--target armv7-linux-androideabi` if they want to continue targeting `armeabi-v7a` ABI.